parser/src/semantic/10_validate_annotations.ts

```diff
@@ -44,6 +45,7 @@ const REST_ENCODABLE_TYPES: Function[] = [
     CnpjPrimitiveType,
     UuidPrimitiveType,
     HexPrimitiveType,
+    HtmlPrimitiveType,
```

Comment:
This array should only have types that are encodable as path, query or header. I don't think html qualifies.